source/fundamentals/enterprise-authentication.txt

```diff
@@ -2,4 +2,261 @@
 
 ====================================
 Enterprise Authentication Mechanisms
-====================================
\ No newline at end of file
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using the
+**authentication mechanisms** available only in the {+mongo-enterprise+}.
+Authentication mechanisms are processes by which the driver and server confirm
+the identity of a client to ensure security before connecting.
+
+You can use the following authentication mechanisms with the latest version of
+{+mongo-enterprise+}.
+
+- :ref:`csharp-kerberos`
+- :ref:`csharp-LDAP`
+
+To authenticate using another mechanism, see the
+:ref:`Authentication Mechanisms guide <csharp-authentication-mechanisms>`. For
+more information on establishing a connection to your MongoDB cluster, see the
+:ref:`csharp-connect-to-mongodb`.
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+You can specify your authentication mechanism and credentials when connecting to
+MongoDB using either of the following methods:
+
+- A **connection string**, also known as a **connection URI**, which is a string
+  that tells the driver how to connect to a MongoDB deployment and how to behave while
+  connected.
+
+- A factory method for the supported authentication mechanism, contained in the 
+  ``MongoCredential`` class.
+.. _csharp-kerberos:
+
+GSSAPI/Kerberos
+---------------
+
+The Generic Security Services API (``GSSAPI``) authentication mechanism allows the 
+user to authenticate to a Kerberos service using the user's principal name.
+
+The following examples specify the authentication mechanism using the following
+placeholders:
+
+- ``<username>``: Your URL-encoded principal name, e.g. "username%40REALM.ME"
```

Comment:
Issue: [Try to avoid "e.g."](https://www.mongodb.com/docs/meta/style-guide/terminology/general-term-guidelines/avoid-obscure-words/) (this trips me up too)

Suggestion: 
```suggestion
- ``<username>``: Your URL-encoded principal name; for example, "username%40REALM.ME"
```